core/lib/presentation/views/dialog/confirmation_dialog_builder.dart

```diff
@@ -233,6 +238,11 @@ class ConfirmDialogBuilder {
                   ),
                 ),
               ),
+            if (_additionalWidgetContent != null)
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
```

Comment:
_textContent and _additionalWidgetContent also have padding, make a lot of space between it. No need `top = 8` here.

Reply:
done !